fury/io.py

```diff
@@ -6,6 +6,9 @@
 from fury.utils import set_input
 
 
+SURFACE_FORMAT = [".vtk", ".vtp", ".fib", ".ply", ".stl", ".xml", ".obj"]
```

Comment:
What is the goal of this variable? I do not see any use

Reply:
I feel this could be usefull later on for Dipy (or other outside package) to know what format is supported,
But for now it's not used, It's more a suggestion questionning.

Maybe there is also a better way to do it, with a function or ...

Reply:
Indeed, I prefer a function. It will avoid mistakes because we could get all extensions via the dictionary below and avoid duplication errors. I always prefer when we can change things in 1 place only.

Can you fix it or you prefer me to merge this PR and I will create a new PR?

Reply:
I simply removed it for now, maybe will create a new issue / PR when needed

Reply:
ok, sounds good. Waiting for the CI's